src/main.rs

```diff
@@ -30,6 +32,10 @@ fn initialize(args: &Args) -> Result<Context, IxaError> {
         .clone();
     // model tidyness -- random seed, automatic shutdown
     context.init_random(parameters.seed);
+
+    // load the population from person record in input file
+    population_loader::init(&mut context)?;
```

Comment:
Perhaps population_loader should be a trait extension on Context?

Reply:
We may want to keep this separate module as a safety feature, to keep anyone from accessing the public extension outside of main and initializing the population from another crate

Reply:
I don't have a strong opinion on if it should be a trait extension (@k88hudson-cfa , wdyt), but I think you can just do `pub(crate)` for the trait, no?